### DISCUSSION TITLE
NEW FEED - Let me disable it

### DISCUSSION THREAD

**Discussion by 0xpr03** [community/community#53782] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm using github to be productive. There are "discover" sites, you have the right panel for "interesting projects".

But don't give me another ADHD inducing "look what other people did" twitter/facebook/mastodon/.. feed. This is not the place for that. I only want to see what happens to my projects, thanks.

**Comment by tvdijen** [comment]
Same feedback I gave to the beta feature, but they still push it
